### ISSUE TITLE
Should exporting with Z=Up be supported?

### ISSUE THREAD

**Issue by donmccurdy** [KhronosGroup/glTF-Blender-Exporter#134]
A recent PR (#130) contributed this, I just wanted to check that this should be supported? In other words, should we have options that are technically invalid according to the spec, and/or should it be named something other than *Convert Z up to Y up*?

**Comment by donmccurdy** [comment]
in past discussions of the three.js Blender exporter, this option was removed: https://github.com/mrdoob/three.js/issues/11803

**Comment by Nehon** [comment]
I agree this should not be part of the exporter, it produces invalid gltf files.
If switching axes is needed it should be the client importer's responsibility

**Comment by donmccurdy** [comment]
@UX3D-nopper I think maybe you've closed this assuming I was suggesting that we add the option, which now exists. I am actually suggesting that we remove the option, as it produces invalid glTF files and may cause loaders to expect this.

If we choose to remove the option it would be good to investigate https://github.com/KhronosGroup/glTF-Blender-Exporter/issues/154 first, although I'm guessing that is not an exporter problem at all.
